reference/features/security.md

```diff
@@ -0,0 +1,272 @@
+
+[`GET /keys/:key`](/reference/api/keys.md#get-one-key), where `:key` stands for the full value of an active key, returns information on a single key.
+
+#### Example
+
+You can query your instance to confirm which active keys have access to your `scifi_books` index:
+
+<CodeSamples id="security_guide_list_keys_1" />
+
+```json
+{
+  "results": [
+    {
+      "description": "Default Search API Key (Use it to search from the frontend code)",
+      "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+      "actions": [
+          "search"
+      ],
+      "indexes": [
+          "*"
+      ],
+      "expiresAt": null,
+      "createdAt": "2021-08-11T10:00:00Z",
+      "updatedAt": "2021-08-11T10:00:00Z"
+    },
+    {
+      "description": "Default Admin API Key (Use it for all other operations. Caution! Do not share it on the client side)",
+      "key": "380689dd379232519a54d15935750cc7625620a2ea2fc06907cb40ba5b421b6f",
+      "actions": [
+          "*"
+      ],
+      "indexes": [
+          "*"
+      ],
+      "expiresAt": null,
+      "createdAt": "2021-08-11T10:00:00Z",
+      "updatedAt": "2021-08-11T10:00:00Z"
+    }
+  ]
+}
+```
+
+### Updating an API key
+
+You can freely update an API key until it expires. This includes editing the indexes, endpoints, and routes it can access, as well as its description and expiry date.
+
+To update an API key, you must use the [`PATCH /keys/:key`](/reference/api/keys.md#update-a-key) endpoint which can only be accessed with the `master` key. Here, `:key` stands for the full value of the key you want to update.
+
+#### Example
+
+After creating your `Manage sci-fi books key` with an expiry date set for the next day, you decide to extend its duration in case you need to update some of the books:
+
+<CodeSamples id="security_guide_update_key_1" />
+
+```json
+{
+  "description": "Manage sci-fi books key",
+  "key": "d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4",
+  "actions": [
+    "documents.add",
+    "documents.get",
+    "documents.delete"
+  ],
+  "indexes": [
+  "scifi_books"
+  ],
+  "expiresAt": "2022-03-01T00:00:00Z",
+  "createdAt": "2022-01-01T10:00:00Z",
+  "updatedAt": "2022-01-01T10:00:00Z"
+}
+```
+
+As you can see, MeiliSearch supports partial updates with the `PATCH` route. This means that your payload can contain only the data you want to update—in this case, `expiresAt`.
+
+### Deleting an API key
+
+If a key is no longer useful or has been compromised, you can use [`DELETE /keys/:key`](/reference/api/keys.md#delete-a-key) to disable it before its expiry date.
+
+#### Example
+
+Since you don't expect to routinely update your instance's indexes after adding the first batch of documents, you decide the default admin key MeiliSearch is not necessary:
```

Comment:
```suggestion
Since you don't expect to routinely update your instance's indexes after adding the first batch of documents, you decide the default admin key is not necessary:
```